modules/tensor_mechanics/include/actions/DomainIntegralAction.h

```diff
@@ -143,4 +151,6 @@ class DomainIntegralAction : public Action
   bool _solid_mechanics;
   /// Whether the constitutive models for the mechanics calculations use an incremental form
   bool _incremental;
+  /// Exponent on the effective stress
```

Comment:
Add 'used by the C integral' to this comment.

Reply:
Should be done